.golangci.yml

```diff
@@ -0,0 +1,91 @@
+run:
+  concurrency: 4
+  timeout: 1m
+
+  # exit code when at least one issue was found
+  issues-exit-code: 1
```

Comment:
Do you want to fail the step when at least one issue is found? It looks like golangci-lint's own repo uses exit code 0 ([link](https://github.com/golangci/golangci-lint-action/blob/125ac487c1b74432b4cd1fb787b5f7439be0c6da/.github/workflows/test.yml#L30))

Reply:
Yes.  I think they are using a 0 exit code because I think their test has linter errors that they constitute "success".  In our case, I want linter errors to present as a failure that needs fixing.  (Maybe I'm misunderstanding your suggestion?)

Reply:
Thanks. Yes, I thought about and I think you are right.